core/services/synchronization/explorer_client_test.go

```diff
@@ -177,16 +177,14 @@ func TestWebSocketClient_Status_ConnectAndServerDisconnect(t *testing.T) {
 	})
 	gomega.NewWithT(t).Eventually(func() synchronization.ConnectionStatus {
 		return explorerClient.Status()
-	}).Should(gomega.Equal(synchronization.ConnectionStatusConnected))
+	}, 5 * time.Second, 100*time.Millisecond).Should(gomega.Equal(synchronization.ConnectionStatusConnected))
```

Comment:
This should already be using the default set in cltest of 30s: https://github.com/smartcontractkit/chainlink/blob/develop/core/internal/cltest/cltest.go#L113
Is that not being picked up?

Reply:
Oh didn't know we set that, interesting. Must have been a timeout in CallbackOrTimeout then - I bumped that